configs/votenet/README.md

```diff
@@ -38,3 +38,13 @@ Then you can use the converted checkpoints following [getting_started.md](../../
 ## Indeterminism
 
 Since test data preparation randomly downsamples the points, and the test script uses fixed random seeds while the random seeds of validation in training are not fixed, the test results may be slightly different from the results reported above.
+
+## IoU loss
+
+Adding IoU loss (simply =1-IoU) boosts VoteNet's performance.
+
+|  Backbone   | Lr schd | Mem (GB) | Inf time (fps) | AP@0.25 |AP@0.5| Download |
+| :---------: | :-----: | :------: | :------------: | :----: |:----: | :------: |
+|    [PointNet++](./votenet_iouloss_8x8_scannet-3d-18class.py)     |  3x    |4.1||63.81|44.21|/|
+
+Please refer to [votenet_iouloss_8x8_scannet-3d-18class.py](./votenet_iouloss_8x8_scannet-3d-18class.py) for the added loss term. Currently we only support calculating IoU loss for axis-aligned bounding boxes since the general IoU calculation CUDA op doesn't support a backward method. Therefore IoU loss is correctly implemented for ScanNet dataset only.
```

Comment:
1. Simply add a config of that loss for usage in this readme.
2. rephrase
```markdown
For now, we only support calculating IoU loss for axis-aligned bounding boxes since the CUDA op of general 3D IoU calculation does not implement the backward method. Therefore, IoU loss can only be used for ScanNet dataset for now.
```